app/controllers/application_controller/ci_processing.rb

```diff
@@ -560,7 +560,7 @@ def live_migrate_vm
   def live_migrate_finished
     task_id = session[:async][:params][:task_id]
     vm_id = session[:async][:params][:id]
-    vm = VmOrTemplate.find_by(:id => vm_id)
+    vm = find_by_id_filtered(VmOrTemplate, vm_id)
```

Comment:
`find_by_id_filtered` method was renamed to `find_record_with_rbac` (https://github.com/ManageIQ/manageiq-ui-classic/pull/779/commits/dad34557eb21d21317126a51f30f9384197c0ea4)